contracts/transmuter/facets/Redeemer.sol

```diff
@@ -107,10 +109,19 @@ contract Redeemer is IRedeemer {
         address[] memory forfeitTokens
     ) internal returns (address[] memory tokens, uint256[] memory amounts) {
         TransmuterStorage storage ts = s.transmuterStorage();
+        // Reentrant protection
```

Comment:
Maybe we can remove the comments about reentrancy checks as they're duplicates in multiple places

Reply:
or even define a modifier